components/dashboard/queries.ts

```diff
@@ -1,11 +1,13 @@
 import { gql } from '../../lib/graphql/helpers';
 
 import { accountNavbarFieldsFragment } from '../collective-navbar/fragments';
+import { editAccountFragment } from '../edit-collective/sections/Info';
 
 export const adminPanelQuery = gql`
   query Dashboard($slug: String!) {
     account(slug: $slug) {
       id
+      ...EditAccountFragment
```

Comment:
This PR seems like the perfect place to resolve https://github.com/opencollective/opencollective/issues/7951 and avoid fetching many unneeded fields in the main query.